WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2020,3 +2020,17 @@ ChooserListViewCustomStyleColumn        = @FixBoxModel@; border:0; @TransparentB
 ChooserListViewCustomStyleHeader        = @ChooserListViewCustomStyleColumn@;
 ChooserListViewCustomStyleHeaderLabel   = @FontSize-slightly-bigger@; @Text@; text-transform:none; line-height:1; padding: 0 0 @SkinWrapperPadding@ 0;
 ChooserListViewCustomStyleContainer     = @PanelBorder@; @FixBoxModel@;
+
+#######################################################################
+#   Task List view
+#######################################################################
+TaskListViewAddNewTaskIcon               = display:inline-block; margin-right: 12px;
+TaskListViewNewTaskBannerWrapper         = @Text-hint@ @InactiveCursor@ @FontSize-big@ padding:16px 0 16px 0; margin-left:6px;
```

Comment:
margin-left is pushing the dynamically created input field to the left by 6px. padding-left should be used here.
ZmTaskListView.prototype._resetInputSize() function calculating the input position based on the div position which should not be changed .